src/ShapeFile.js

```diff
@@ -1,23 +1,19 @@
-import React, {PropTypes}  from 'react';
-import {Path} from 'react-leaflet';
-import Shapefile from './leaflet.shpfile';
+import React, { useEffect, useState } from 'react';
+import { GeoJSON } from 'react-leaflet'
+import shp from 'shpjs'
+import "babel-polyfill"
```

Comment:
Do we use this?

Reply:
yes

Reply:
not anymore if you refere to babel I will delete it